exporter/azure_monitor/common/common.go

```diff
@@ -0,0 +1,65 @@
+package common
+
+import (
+	"context" 
+)
+
+// Some structs will be used in future PRs
+
+type AzureMonitorContext struct {
+    sdkVersion ai_internal_sdkVersion
+}
+
+type ai_internal_sdkVersion struct {
+    goPlatform_version string
+    opencensus_version string
+    ext_version string
+}
+
+type BaseObject struct { // Used to avoid repeat attributes
+	ver int
+	Name string
+	time string
+	sampleRate int
+	success bool
+	IKey string
+	tags string
+	data string
+}
+
+type Options struct {
+	ServiceName        	string

```

Comment:
How come some of these field names are CamelCase and others are not

Reply:
I originally wanted them to all be lowercase, but golang requires the attribute to start with a capital letter to be public. I don't know yet if I want everything to be public, so I am only making attributes public as I go along. Definitely something that will change as I proceed.